frontend/config-assistant/src/components/CombinedEditorComponent.vue

```diff
@@ -17,7 +16,10 @@ import {useToast} from 'primevue/usetoast';
 import {useConfirm} from 'primevue/useconfirm';
 import {confirmationService} from '@/helpers/confirmationService';
 import {toastService} from '@/helpers/toastService';
-
+import RadioButton from 'primevue/radiobutton';
+import Dialog from 'primevue/dialog';
```

Comment:
are all those imports necessary?

Reply:
yes it is necessary.

Reply:
I checked out the branch for me there all greyed out: 
![image](https://github.com/PaulBredl/config-assistant/assets/34026908/2b12b10d-0fd2-43cb-9c89-4095316b53c8)

Reply:
its not anymore now @PaulBredl